### PR TITLE
refactor(accordion): From module.css to /package.css #1846

### PR DESCRIPTION
**Pull request by poi33**
Part of #1846

New accodion.css with BEM naming
Refactored from Accordion.module.css

### CODE REVIEW

**Comment by poi33** [comment]
Do we want the `open` modifier to be part of the `.accordion` block: `.accordion--open` 
Or closer to its implementation of `.accordion__item`  element: `.accordion__item--open` ?

File: packages/react/src/components/Accordion/AccordionHeader/AccordionHeader.tsx [hunk 1]
```diff
@@ -39,19 +38,19 @@ export const AccordionHeader = forwardRef<
...
-        className={cl(classes.accordionButton, `fds-focus`)}
+        className={cl('accordion__button', `fds-focus`)}
         onClick={handleClick}
         aria-expanded={context.open}
         aria-controls={context.contentId}
       >
         <ChevronDownIcon
           aria-hidden
-          className={classes.expandIcon}
+          className={cl('accordion__expand-icon')}
```
Comment (mimarz) [inline packages/react/src/components/Accordion/AccordionHeader/AccordionHeader.tsx:53]: Suggest removing redundant use of `clsx` if its not needed :)
```suggestion
          className='accordion__expand-icon'
```

**Review by mimarz — CHANGES_REQUESTED** [review]
Awesome work!

I like that you took the time to clean up the css as well. Some components CSS are better than other, so its also something we do whenever we see the need 🙌

Two things I forgot to mention in the "main" issue:
- Prefix the new classes with `fds-`
- Make sure to run `build` in the root (or `packages/react`) to generate a new `react-css-modules.css` which excludes the now removed css modules css :) 

(I've also updated the main issue with these points)

**Review by mimarz — APPROVED** [review]
LGTM 👍 

Thank you!

**Pull request merged**